### PR TITLE
Weather image

### PR DESCRIPTION
**Pull request by Todd-Estes**
In this PR:

**-Added weather image functionality to destination forecasts on index page**

**-Created select_weather_image to Application Controller** _(May need refactoring)_

**-Updated index spec**

- [x] Updated Files: application_controller, flight index spec, index view

- [x] Repo Representation: Front End

- [x] Happy Path Tested

- [x] All Tests are passing

- [x] Collaborators: Will Dunlap, Todd Estes

### CODE REVIEW

**Review by cpfergus1 — CHANGES_REQUESTED** [review]
Shouldn't this method belong in the PORO? 
looks great but I don't think application controller is the place for this

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+      when snowy_weather
+        "weather_images/snow.jpg"
+      when partly_sunny_weather
+        "weather_images/partly_sunny.jpg"
+      when thunder_storms_weather
+        "weather_images/thunder_storms.jpg"
+      else
+        "weather_images/partly_sunny.jpg"
+    end
+  end
```
Comment (JesseMellinger) [inline app/controllers/application_controller.rb:31]: Nice use of a `case` statement to return different images!

**Review by JesseMellinger — COMMENTED** [review]
I'm inclined to agree with Connor on this. If the `select_weather_image` method was needed in other controllers then I think `ApplicationController` would be a good spot, but this method is probably best located as an instance method on the `Trip` poro.

**Review by TeknoServal — CHANGES_REQUESTED** [review]
With connor, the image selector method should be in the poro

**Comment by dunlapww** [comment]
Yeah, like the idea of moving it in the poro.  We were considering a few options, but poro makes a lot of sense.  Will do, and then we'll repush.  thanks for feedack!

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (iandouglas) [reply app/controllers/application_controller.rb:31]: Aside from "snowy" which goes to "snow", you could almost reduce this to "remove '_weather' and add '.jpg'" and do away with the case statement completely

**Pull request closed without merging**